lms/djangoapps/static_template_view/views.py

```diff
@@ -97,3 +106,41 @@ def render_404(request):
 @fix_crum_request
 def render_500(request):
     return HttpResponseServerError(render_to_string('static_templates/server-error.html', {}, request=request))
+
+@ensure_csrf_cookie
+def render_no_cache(request, template):
```

Comment:
Proponuję zmienić tego potworka na zwykły widok FormView. Będzie ładniej i przyjemniej

Reply:
@nazgut możesz się odnieść to tego ? bo to Twoje :)